lib/matplotlib/rcsetup.py

```diff
@@ -1097,7 +1097,8 @@ def validate_hist_bins(s):
     'date.autoformatter.day': ['%Y-%m-%d', six.text_type],
     'date.autoformatter.hour': ['%H:%M', six.text_type],
     'date.autoformatter.minute': ['%H:%M:%S', six.text_type],
-    'date.autoformatter.second': ['%H:%M:%S.%f', six.text_type],
+    'date.autoformatter.second': ['%H:%M:%S', six.text_type],
+    'date.autoformatter.musecond': ['%H:%M:%S.%f', six.text_type],
```

Comment:
Maybe this is the best we can do, at least for now, but jumping from second scale to microsecond scale is quite a leap. I guess the idea is that it is really not for microseconds, but for anything smaller than one second.  At that scale, though, it probably doesn't make sense to carry along the hour, and in some cases not even the minute.